protected/models/cron/cron.vo.php

```diff
@@ -107,94 +58,53 @@ public function getStatus() : string
-     * @param string $action SCron Job Action
-     *
-     * @return bool Is Cron Job Execution Action Set
-     */
-    public function setAction(string $action = '') : bool
+    public function setAction(?string $action = null): void
     {
-        return $this->set('action', $action);
+        $this->set('action', $action);
     }
 
-    /**
-     * Set Cron Job Execution Interval
-     *
-     * @param int $interval Cron Job Execution Interval
-     *
-     * @return bool Is Cron Job Execution Interval Set
-     */
-    public function setInterval(int $interval = -1) : bool
+    public function setInterval(?int $interval = null): void
     {
-        return $this->set('interval', $interval);
+        $this->set('interval', (int) $interval);
     }
 
-    /**
-     * Set Cron Job Next Execution Time
-     *
-     * @return bool Is Cron Job Next Execution Time Set
-     */
-    public function setTimeNextExec() : bool
+    public function setTimeNextExec(): void
     {
         $timeNextExec = time() + $this->getInterval();
 
-        return $this->set('time_next_exec', $timeNextExec);
+        $this->set('time_next_exec', $timeNextExec);
     }
 
-    /**
-     * Set Cron Job Last Execution Status
-     *
-     * @param int $statusCron Job Last Execution Status
-     *
-     * @return bool Is Cron Job Last Execution Status Set
-     */
-    public function setLastExecStatus(int $status = 0) : bool
+    public function setLastExecStatus(int $status = 0): void
     {
-        return $this->set('last_exec_status', $status);
+        $this->set('last_exec_status', $status);
     }
 
-    /**
-     * Set Cron Job Activation Status
-     *
-     * @param int $isActive Cron Job Activation Status
-     *
-     * @return bool Is Cron Job Activation Status Set
-     */
-    public function setIsActive(int $isActive = 0) : bool
+    public function setIsActive(int $isActive = 0): void
     {
-        return $this->set('is_active', $isActive);
+        $this->set('is_active', $isActive);
     }
 
-    /**
-     * Set Cron Job Error Message
-     *
-     * @param string $errorMessage Cron Job Error Message
-     *
-     * @return bool Is Cron Job Error Message Set
-     */
-    public function setErrorMessage(string $errorMessage = '') : bool
+    public function setErrorMessage(?string $errorMessage = null): bool
     {
-        if (strlen(trim($errorMessage)) < 1) {
-            return $this->set('error_message', '');
+        $errorMessage = (string) $errorMessage;
+
+        if (empty(trim($errorMessage))) {
+            $this->set('error_message', null);
+            return false;
         }
 
         if (strlen($errorMessage) > 255) {
-            $errorMessage = mb_substr($errorMessage, 0, 254).'…';
+            $errorMessage = mb_substr($errorMessage, 0, 254);
+            $errorMessage = sprintf('%s…', $errorMessage);
         }
 
-        return $this->set('error_message', $errorMessage);
+        $this->set('error_message', $errorMessage);
```

Comment:
Codacy found an issue: [Variable "$errorMessage" is not in valid snake_case format, try "$error_message"](https://app.codacy.com/manual/common/buryak/pullRequest?prid=5800482)